pkg/agent/dispatcher.go

```diff
@@ -302,7 +323,12 @@ func (disp *Dispatcher) send(ctx context.Context, logger *zap.Logger, id uint64,
 	// by base64 encoding it, so use RawMessage to avoid serializing to []byte
 	// (done by SerializeMonitorMessage), and then base64 encoding again
 	raw := json.RawMessage(data)
-	logger.Info("sending message to monitor", zap.ByteString("message", raw))
+	if bytes.HasPrefix(data, []byte("{\"content\":{},\"type\":\"HealthCheck\",\"id\":")) {
+		// don't log health checks, they're too frequent
+		logger.Debug("sending message to monitor", zap.ByteString("message", raw))
+	} else {
+		logger.Info("sending message to monitor", zap.ByteString("message", raw))
+	}
```

Comment:
IMO it's ok to downgrade this to debug in all cases. Any messages we send that aren't health checks will already have log lines from `exec_monitor.go`